src/libs/state.ts

```diff
@@ -15,11 +18,16 @@ export const authStore = Utils.atom<any>({
   fenceStatus: {},
   cookiesAccepted: undefined,
   authContext: undefined,
-  oidcConfig: {},
+  oidcConfig: {
+    authorityEndpoint: undefined,
+    clientId: undefined,
+  },
   isAzurePreviewUser: undefined,
+  sessionId: undefined,
```

Comment:
So I understand -- `sessionId` is a unique identifier for the user's session starting with an explicit login. Changes to renew the access token via the refresh token will not update the `sessionId`. Is that right?

Reply:
That's correct, it currently only updates during a `signIn`